checker/build.gradle

```diff
@@ -51,22 +60,38 @@ task copyJarsOtherThanJdk8ToDist(dependsOn: shadowJar, group: 'Build') {
         }
     }
 }
+boolean shouldForceBuildJdk = false
```

Comment:
Who would use this target?  (I'm wondering whether it is necessary.)
Also, a cleaner way to achieve the goal is to just delete jdk.jar and then run buildJdk.  That doesn't require any `shouldForceBuildJdk` or any change to the `buildJdk` task.